libdnf/system/state.hpp

```diff
@@ -103,6 +103,10 @@ class State {
     /// @since 5.0
     State(const std::filesystem::path & path);
 
+    /// @return True if the State instance does not contain information about packages
+    /// @since 5.0
```

Comment:
Is it a test to trigger conversion? May I ask you add a purpose of the method to the description?

Reply:
If it is a trigger then what about to detect the situation by presence of system state files?